install_config/topics/installing_with_bastion_vm.adoc

```diff
@@ -0,0 +1,279 @@
+vm_infra_wait_for_ip_delay: 20
+
+node_item: &node_item
+  cluster: "{{ openshift_ovirt_cluster }}"
+  template: "{{ template_name }}"
+  memory: "8GiB"
+  cores: "2"
+  high_availability: true
+  disks:
+    - name: docker
+      size: 15GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+    - name: openshift
+      size: 30GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+  state: running
+  cloud_init:
+    root_password: "{{ root_password }}"
+    authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+    custom_script: "{{ cloud_init_script_node | to_nice_yaml }}"
+
+openshift_ovirt_vm_profile:
+  master_vm:
+    <<: *node_item
+    memory: 16GiB
+    cores: "{{ vm_cores | default(4) }}"
+    disks:
+      - name: docker
+        size: 15GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: openshift_local
+        size: 30GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: etcd
+        size: 25GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+    cloud_init:
+      root_password: "{{ root_password }}"
+      authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+      custom_script: "{{ cloud_init_script_master | to_nice_yaml }}"
+  node_vm:
+    <<: *node_item
+  etcd_vm:
+    <<: *node_item
+  lb_vm:
+    <<: *node_item
+
+cloud_init_script_node: &cloud_init_script_node
+  packages:
+    - ovirt-guest-agent
+  runcmd:
+    - sed -i 's/# ignored_nics =.*/ignored_nics = docker0 tun0 /' /etc/ovirt-guest-agent.conf
+    - systemctl enable ovirt-guest-agent
+    - systemctl start ovirt-guest-agent
+    - mkdir -p /var/lib/docker
+    - mkdir -p /var/lib/origin/openshift.local.volumes
+    - /usr/sbin/mkfs.xfs -L dockerlv /dev/vdb
+    - /usr/sbin/mkfs.xfs -L ocplv /dev/vdc
+  mounts:
+    - [ '/dev/vdb', '/var/lib/docker', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdc', '/var/lib/origin/openshift.local.volumes', 'xfs', 'defaults,gquota' ]
+  power_state:
+    mode: reboot
+    message: cloud init finished - boot and install openshift
+    condition: True
+cloud_init_script_master:
+  <<: *cloud_init_script_node
+  runcmd:
+    - sed -i 's/# ignored_nics =.*/ignored_nics = docker0 tun0 /' /etc/ovirt-guest-agent.conf
+    - systemctl enable ovirt-guest-agent
+    - systemctl start ovirt-guest-agent
+    - mkdir -p /var/lib/docker
+    - mkdir -p /var/lib/origin/openshift.local.volumes
+    - mkdir -p /var/lib/etcd
+    - /usr/sbin/mkfs.xfs -L dockerlv /dev/vdb
+    - /usr/sbin/mkfs.xfs -L ocplv /dev/vdc
+    - /usr/sbin/mkfs.xfs -L etcdlv /dev/vdd
+  mounts:
+    - [ '/dev/vdb', '/var/lib/docker', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdc', '/var/lib/origin/openshift.local.volumes', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdd', '/var/lib/etcd', 'xfs', 'defaults,gquota' ]
+----
+
+<1> FQDN of the Manager machine.
```

Comment:
Unless you just need to change these two parameters, you have more to annotate and explain.

Reply:
Only two parameters need to be changed.